scripts/resolve_bugs.pl

```diff
@@ -42,8 +44,17 @@
 $auto_user->{bless_groups} = [ Bugzilla::Group->get_all ];
 Bugzilla->set_user($auto_user);
 
-my $query = { resolution => '---', product => $product };
-$query->{component} = $component if defined $component;
+if ($buglist) {
+    my $uri = URI->new($buglist);
+    die "The buglist url must match bugzilla.mozilla.org" if $uri->host ne 'bugzilla.mozilla.org';
+    die "Path must point to /buglist.cgi" if $uri->path ne '/buglist.cgi';
+    die "Protocol must match https" if $uri->protocol ne 'https';
+    $query = { $uri->query_form };
+}
+else {
+    $query = { resolution => '---',};
```

Comment:
should include the product name too, { resolution => '---', product => $product }